content/misc/glossary/contents.lr

```diff
@@ -347,7 +382,8 @@ A publicly-listed node in the [Tor network](#tor-/-tor-network/-core-tor) that f
 
 ### Satori
 
-It is an [add-on](#add-on-extension-or-plugin) for the Chrome or Chromium browsers that allows you to download several security and privacy programs, including [Tor Browser](#tor-browser), from different sources. You can Install Satori from the Chrome Web Store.
+It is an [add-on](#add-on-extension-or-plugin) for the Chrome or Chromium browsers that allows you to download several security and privacy programs, including [Tor Browser](#tor-browser), from different sources. 
+You can Install Satori from the Chrome Web Store.
```

Comment:
I think Satori isn't maintained anymore. So we should update this or remove the installation part.